src/components/Global/ImageGrid.vue

```diff
@@ -71,8 +61,8 @@ watch(() => props.images, () => {
       :cols="columnSpan"
     >
       <v-img
-        v-if="image.basename in imageDetails"
-        :src="imageDetails[image.basename].url"
+        v-if="image.basename in imageDetails && imageDetails[image.basename]"
```

Comment:
I'm going to leave it as-is... partly because lodash isn't a pulled into that component yet, and partly because I'm not sure if lodash isNil checks for empty string within a ref...